src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -25,37 +31,53 @@ object LocalBackend {
 }
 
 
+/**
+ * Handles both local Docker runs as well as local direct command line executions.
+ * Or at least it will.
+ */
 class LocalBackend extends Backend {
-
   /**
    * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
-  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
+  override def executeCommand(commandLine: String, workflowDescriptor: WorkflowDescriptor, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
     import LocalBackend._
 
-    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout")
-    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr")
-    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command")
+    val callDirectory = Paths.get(hostExecutionPath(workflowDescriptor).toFile.getAbsolutePath, "call-" + call.name).toFile
+    callDirectory.mkdir()
+
+    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout", callDirectory)
+    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr", callDirectory)
+    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command", callDirectory)
 
     commandWriter.write(commandLine)
     commandWriter.flushAndClose()
 
-    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline)
+    // FIXME this is hardcoded to use Docker for the very short term.  Once we can determine Docker/non-Docker
+    // FIXME then dockerRun can either be what it is here/empty string and hopefully this should all work.
+    val DockerImage = "ubuntu:latest"
+    val hostWorkflowExecutionFile = hostExecutionPath(workflowDescriptor).toFile
+    val containerExecutionDir = s"/root/${hostWorkflowExecutionFile.toString}"
+    // -v maps the host workflow executions directory to /root on the container.
+    // -w sets the command's working directory on the container.
+    // -i makes the run interactive, required for the cat and <&0 shenanigans that follow.
+    val dockerRun =
+      s"docker run -v ${hostWorkflowExecutionFile.getAbsolutePath}:$containerExecutionDir -w $containerExecutionDir -i $DockerImage"
```

Comment:
An example docker dockerRun command would be:

```
docker run -v $PWD/cromwell-executions/some-workflow-name/0f00-ba4/:/root -w /root/cromwell-executions/some-workflow-name/0f00-ba4/ -i ubuntu:latest
```

My only objection is that I'm not a huge fan of the choice of value for `-w` to Docker.  Why did you choose that?  I was going to object to mounting in the whole workflow directory but then I thought about it a little more and changed my mind and also was further comforted by the fact that this is the "LocalBackend" anyway, so we can make assumptions about things being local.